lightning/src/ln/channel.rs

```diff
@@ -713,6 +713,7 @@ pub(super) enum ChannelError {
 	Ignore(String),
 	Warn(String),
 	Close((String, ClosureReason)),
+	NotFound(String),
```

Comment:
IIUC `ChannelError` represents the different actions that we want to take on an error, so maybe `SendError` would be a better name?